apps/admin_app/lib/admin_app_web/controllers/order_controller.ex

```diff
@@ -37,6 +37,52 @@ defmodule AdminAppWeb.OrderController do
     redirect(conn, to: order_path(conn, :show, order.id))
   end
 
+  def show_invoice(conn, params) do
+    order =
+      load_order(%{number: params["number"]})
+      |> Snitch.Repo.preload(:line_items)
```

Comment:
We can preload many things at once refer [Repo.preload](https://hexdocs.pm/ecto/Ecto.Repo.html#c:preload/3)